src/github/pull-request.ts

```diff
@@ -38,7 +38,7 @@ export const 	pullRequestWebhookHandler = async (context: WebhookContext, jiraCl
 	const gitHubInstallationClient = await createInstallationClient(gitHubInstallationId, subscription.jiraHost, metrics, context.log, gitHubAppId);
 	const reviews = await getPullRequestReviews(subscription.jiraHost, gitHubInstallationClient, context.payload.repository, pull_request, context.log);
 
-	const jiraPayload: JiraPullRequestBulkSubmitData | undefined = await transformPullRequest(gitHubInstallationClient, pull_request, reviews, context.log);
+	const jiraPayload: JiraPullRequestBulkSubmitData | undefined = await transformPullRequestRest(gitHubInstallationClient, pull_request, reviews, context.log);
```

Comment:
I stole the transformPullRequest name for the new implementation, this is for webhooks which for now, will still be running the original code, will use new transform once it has been proven in prod